lib/gui/index.html

```diff
@@ -49,5 +50,15 @@
         manifest-bind="version"
         os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md"></span>
     </footer>
+
+    <div class="section-loader"
+      ng-controller="StateController as state"
+      ng-class="{
+        isFinish: state.is('success')
+      }">
+      <safe-webview
+        src="'https://etcher.io/success-banner/'"
```

Comment:
So, if somebody _did_ accidentally change it to be only single-quoted or double-quoted, would that be picked up by the CI builds, or would it only be triggered as an error at runtime?